save-common/src/commonMain/kotlin/org/cqfn/save/testsuite/TestSuiteDto.kt

```diff
@@ -0,0 +1,17 @@
+package org.cqfn.save.testsuite
+
+import org.cqfn.save.entities.Project
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property type
+ * @property name
+ * @property project
+ */
+@Serializable
+data class TestSuiteDto(
+    var type: TestSuiteType,
```

Comment:
Why are these fields mutable? I believe for dtos we are not changing them after object creation?